wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/PersonAndroid.kt

```diff
@@ -0,0 +1,129 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source file: person.proto
+package com.squareup.wire.protos.person
+
+import com.squareup.wire.EnumAdapter
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import com.squareup.wire.WireEnum
+import com.squareup.wire.internal.Internal
+import com.squareup.wire.kotlin.UnkownFieldsBuilder
+import kotlin.Int
+import kotlin.String
+import kotlin.collections.List
+import okio.ByteString
+
+import android.os.Parcel
+import android.os.Parcelable
+import com.squareup.wire.kotlin.decodeMessage
+
+data class PersonAndroid(
+    val name: String,
+    val id: Int,
+    val email: String? = null,
+    val phone: List<PhoneNumber> = emptyList(),
+    val unknownFields: ByteString = ByteString.EMPTY
+) : Parcelable {
+
+  override fun writeToParcel(dest: Parcel, p1: Int) = dest.writeByteArray(ADAPTER.encode(this))
+
+  override fun describeContents() = 0
+
+  object CREATOR : Parcelable.Creator<PersonAndroid> {
+    override fun createFromParcel(input: Parcel) = ADAPTER.decode(input.createByteArray())
+
+    override fun newArray(size: Int): Array<PersonAndroid?> = arrayOfNulls(size)
+  }
```

Comment:
@swankjesse / @Egorand  PTAL at this PersonAndroid class which implements Parcelable

Reply:
thumbs up